applications/asset_tracker/CMakeLists.txt

```diff
@@ -22,9 +22,37 @@ zephyr_library_include_directories(
   src/orientation_detector
   src/ui
   src/cloud_codec
+  src/battery_monitor
   )
 
 # Application sources
 add_subdirectory(src/orientation_detector)
 add_subdirectory(src/ui)
 add_subdirectory(src/cloud_codec)
+add_subdirectory(src/gps_controller)
+if (CONFIG_BATTERY_MONITOR)
+add_subdirectory(src/battery_monitor)
+endif()
+
+include_directories(src/env_sensors)
```

Comment:
s/include_directories/zephyr_include_directories/

Reply:
Even better would be to modify line 18, zephyr_include_directories(src), to be

zephyr_include_directories(
  src
  src/env_sensors
)

or possibly have whatever is trying to include things in env_sensors do #include <src/env_sensors/something_in_env_sensors.h>